pkg/errors/errors.go

```diff
@@ -134,6 +134,12 @@ var (
 
 	// ErrManifestFoundButNoDeployCommands raised when a manifest is found but no deploy commands are defined
 	ErrManifestFoundButNoDeployCommands = errors.New("found okteto manifest, but no deploy commands where defined")
+
+	// ErrDeployHasNotDeployAnyResource raised when a deploy command has not created any resource
+	ErrDeployHasNotDeployAnyResource = UserError{
+		E:    errors.New("It seems that you haven't deployed anything"),
```

Comment:
we should have two versions of this error, one for deployment errors, another one if you didn't deploy anything. But the hint should be the same.

Reply:
I think that you can have a deployment in an error status and debug it by running `okteto up`. Does this makes sense?

Reply:
Note that it's not the same that the deploy command fails, than the container is failing later after deployment.
if the deploy fails I think it's fine to ask to fix the deploy commands